### ISSUE TITLE
PHP Undefined index: type Error

### ISSUE THREAD

**Issue by d-karstens** [verbb/vizy#18]
**Description**
We've been trying out Vizy and it seems like a pretty great plugin to solve some headaches we've been having with content authoring 👌

Unfortunately we've been running into the following issue when saving re-saving an entry when using Vizy inside matrix or super table fields

![Screen Shot 2021-05-12 at 3 41 38 pm](https://user-images.githubusercontent.com/21274925/117926886-98ead780-b338-11eb-9493-e5f906533130.png)

![Screen Shot 2021-05-12 at 3 34 10 pm](https://user-images.githubusercontent.com/21274925/117926222-9471ef00-b337-11eb-892f-60835d064d2f.png)

I've done a little digging and it seems like this is potentially resolved just with a check for `type` before [here](https://github.com/verbb/vizy/blob/craft-3/src/models/NodeCollection.php#L153), but I'm not sure if something else is going on with the error only happening if the entry has been saved once before

Love your work!

**Additional info**
- Plugin version: 1.01
- Craft version: 3.6.14
- Multi-site: Yes or No

**Additional context**

**Comment by engram-design** [comment]
Yep, this was caused by a Vizy field being nested in a Matrix field (or similar), and when saving the field containing un-changed block content. This is because Craft's delta-handling kicks in, which doesn't play nicely with how Vizy needs to work. Hence, only a tiny portion of the field data was being POST-ed back to Craft, causing this issue.

Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Thank you!

**Comment by engram-design** [comment]
Fixed in [1.0.2](https://github.com/verbb/vizy/releases/tag/1.0.2)
